migrations/sql/V2019.07.24.11.55__add_tables_to_stage_NROS_and_VFCBC_data.sql

```diff
@@ -0,0 +1,495 @@
+CREATE SCHEMA IF NOT EXISTS NOW_Submissions;
+
+CREATE TABLE NOW_Submissions.application_start_stop (
+    MESSAGEID integer PRIMARY KEY,
```

Comment:
So, I see a bunch of these don't have underscores, and others do. Are we leaving them as-is to simplify the ETL process? With the assumption that it doesn't matter if the naming patterns are different because it's in a different schema?

I'm creating the python models, and this stood out right away. Thanks!

Reply:
For this everything was meant to just be the same as it is in the databases we are pulling from. for the most part all of the columns are the same as what they use and we have added a few surrogate PK's on tables that had combo PK's. Half the tables had underscores in the name and the other half didn't so thats the one thing we normalized so they all have underscores now.

Reply:
Regarding underscores, I was referring to the column names. Our column names use underscores, so I would expect this to be called `message_id`, for example. Some of these column names have underscores, and some do not.

Reply:
Yeah, the SEQ_NO is a weird one that was added by NROS and for some reason doesn't match the rest of the naming conventions. They are a mor or less straight copy paste though.